tests/integration/tests/cluster/test-dashboard-version.spec.js

```diff
@@ -8,11 +8,18 @@ context('Test Kyma Dashboard Version link', () => {
   });
 
   it('Check Kyma Dashboard Version link', () => {
-    cy.get('.ui5-sn-divider')
-      .next()
-      .find('ui5-tree-item')
+    cy.get('[title="Profile"]').click();
+
+    cy.get('.ui5-menu-rp')
+      .find('ui5-li:visible')
+      .contains('Legal Information')
+      .invoke('show')
+      .click({ force: true });
+
+    cy.get('ui5-responsive-popover[placement-type="Right"]')
+      .get('ui5-li:visible')
       .last()
-      .should('have.attr', 'title')
-      .and('include', 'Kyma Dashboard version:');
+      .invoke('text')
```

Comment:
same question here for 
`.invoke('show')` and `.invoke('text')`

Reply:
`.invoke('show')` can be removed but `.invoke('text')` is necessary to get Kyma Dashboard version: as string